tests/testthat/payload/model_submit_payload.json

```diff
@@ -21,7 +21,16 @@
     "anc_prevalence_year2": 2018,
     "anc_art_coverage_year1": 2016,
     "anc_art_coverage_year2": 2018,
-    "no_of_samples": 20
+    "spectrum_population_calibration": "national",
+    "spectrum_plhiv_calibration_level": "subnational",
+    "spectrum_plhiv_calibration_strat": "sex_age_group",
+    "spectrum_artnum_calibration_level": "national",
+    "spectrum_artnum_calibration_strat": "age_coarse",
+    "artattend": false,
+    "rng_seed": 17,
+    "no_of_samples": 20,
+    "max_iter": 250,
+    "permissive": false
```

Comment:
does`<version_info>` below need updating? Are there any others that do too?

Reply:
I replace this as a whole thing in the test setup using `gsub` so slightly different mechanism

Reply:
ok cool - would be nice if json allowed comments as then we could leave a note to that effect :)